drivers/bridge/bridge.go

```diff
@@ -55,14 +55,15 @@ type configuration struct {
 
 // networkConfiguration for network specific configuration
 type networkConfiguration struct {
-	ID                 string
-	BridgeName         string
-	EnableIPv6         bool
-	EnableIPMasquerade bool
-	EnableICC          bool
-	Mtu                int
-	DefaultBindingIP   net.IP
-	DefaultBridge      bool
+	ID                       string
+	BridgeName               string
+	EnableIPv6               bool
+	EnableIPMasquerade       bool
+	EnableICC                bool
+	Mtu                      int
+	DefaultBindingIP         net.IP
+	DefaultBridge            bool
+	ContainerInterfacePrefix string
```

Comment:
As this is internal, please rename to ContainerIfacePrefix so that we can reduce this change's footprint.